docs/source/guides/user/chapters/concepts.rst

```diff
@@ -8,6 +8,11 @@ It is important to understand some basic concepts before start using Avocado.
 Test Resolution
 ---------------
 
+.. note:: Some definitions here may be out of date. The current runner (run)
```

Comment:
I think naming/refering to the current runner  as`run` may be a bit misleading... what I mean is that even when using say the `remote` runner, the `run` command will be used.  Right now, this is how the various runners are named:

```$avocado plugins
...
Plugins that run test suites on a job (runners): 
docker  Runs on a Docker (or compatible) container
nrunner *EXPERIMENTAL* nrunner based implementation of job compliant runner
remote  Runs on a remote machine using SSH
runner  The conventional test runner
vm      Runs on a VM using SSH
```

The name `runner` is not a great name though.